tests/visual-regression.spec.ts

```diff
@@ -0,0 +1,43 @@
+import path from 'node:path';
+import { test, expect } from '@grafana/plugin-e2e';
+import dashboardJson from '../provisioning/dashboards/dashboard.json';
+
+const panelScreenshotOptions = {
+  maxDiffPixels: 0,
+  stylePath: path.join(__dirname, 'show-panel-only.css'),
+};
+
+const { title: dashboardTitle, uid: dashboardUid, panels = [] } = dashboardJson;
+
+const dashboardSlug = dashboardTitle.toLowerCase()
+  .replace(/[^a-zA-Z0-9]+/g, ' ')
+  .trim()
+  .replace(' ', '-');
+
+const dashboardUrl = `http://localhost:3000/d/${dashboardUid}/${dashboardSlug}`;
+
+panels
+  .filter(({ title }) => title !== 'Unconfigured panel')
```

Comment:
🪄 ✨ Unfortunately this uses a "magic string" to identify the unconfigured test panel ... maybe there's a better way to filter this one out that's less brittle using panel configuration properties?